resources/views/admin.blade.php

```diff
@@ -146,22 +146,19 @@ function deleteEmployee(id){
                     });
                 });
 
-                let $id = null
 
                 //Edit Employee
+                let $id = null
                 $('#myTable tbody').on('click', 'button[id^="btn-edit"]', function (e) {
                     const $data =  table.row( $(this).parents('tr') ).data();
                     $id  = $data.id
```

Comment:
this naming conventions `$id` and `$data` will make you trouble once you deep dive into the large code base. just skip this comment now. i will explain once you come to that stage

Reply:
Okay 👍